### PR TITLE
CNV#39964: inferFromVolume support for container disks

### PR DESCRIPTION
**Pull request by ousleyp**
Version(s): 4.17 only (RN)

Issue: [CNV-39964](https://issues.redhat.com//browse/CNV-39964)

Link to docs preview: https://83311--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-17-release-notes.html#virt-4-17-infrastructure

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by RoniKishner** [comment]
/lgtm
looks good from QE side

File: virt/release_notes/virt-4-17-release-notes.adoc [hunk 1]
```diff
@@ -65,6 +65,9 @@ This release adds new features and enhancements related to the following compone
 
 * Configuring xref:../../virt/nodes/virt-node-maintenance.adoc#eviction-strategies[VM eviction strategies] for an xref:../../virt/nodes/virt-node-maintenance.adoc#virt-configuring-cluster-eviction-strategy-cli_virt-node-maintenance[entire cluster] is now generally available.
 
+//CNV-39964 GA: inferFromVolume
+* The `inferFromVolume` attribute is now supported for use with container disks. When requested, {VirtProductName} can copy the labels `instancetype.kubevirt.io/default-instancetype` and `instancetype.kubevirt.io/default-preference` from a source container disk to the boot volume of a new VM.
```
Comment (0xFelix) [inline virt/release_notes/virt-4-17-release-notes.adoc:69]:
```suggestion
* The `inferFromVolume` attribute is now supported for use with imported container disks. When requested, {VirtProductName} can copy the labels `instancetype.kubevirt.io/default-instancetype` and `instancetype.kubevirt.io/default-preference` from a source container disk to the boot volume of a new VM.
```

Comment (0xFelix) [reply virt/release_notes/virt-4-17-release-notes.adoc:69]: Very slight but important change. Imported containerdisks can make use of the labels, but containerdisks used as ephemeral volumes can not yet.

Comment (ousleyp) [reply virt/release_notes/virt-4-17-release-notes.adoc]: @0xFelix I made the change :) thanks for pointing that out!

**Review by abhatt-rh — COMMENTED** [review]
LGTM!

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by 0xFelix — APPROVED** [review]
Thanks

/lgtm

**Pull request merged**